backend/controllers/prisonerProfile/prisonerPersonal.ts

```diff
@@ -48,6 +50,19 @@ export default ({
     const treatmentCodes = treatmentTypes && treatmentTypes.map((type) => type.code).join()
     const healthCodes = healthTypes && healthTypes.map((type) => type.code).join()
 
+    const {
+      app: { neurodiversityEnabledUsernames, neurodiversityEnabledPrisons },
+    } = config
+
+    // TODO: Part of temporary measure to only allow restricted prisons access to neurodivergence data. If no prisons specified then assume all are allowed access.
+    const getNeurodivergenceSupportNeed = async () => {
+      let divergence = createFlaggedContent([])
```

Comment:
Why flagged?

Reply:
Ok. I mean it's confusing because i don't see anywhere where the `enabled` part of it is ever referenced.

There is no point having flagged content that is shown without checking the flag?!

I guess this is something we can deal with later though